src/main/java/tools/jackson/databind/cfg/MapperBuilder.java

```diff
@@ -1236,6 +1236,19 @@ public B propertyNamingStrategy(PropertyNamingStrategy s) {
         return _this();
     }
 
+    /**
+     * Method for configuring {@link EnumNamingStrategy} to use for adapting
+     * POJO enum names (internal) into content property names (external)
+     *
+     * @param s Strategy instance to use
+     *
+     * @return Builder instance itself to allow chaining
```

Comment:
add `@since 2.18` to all new public methods (including the ones that you didn't add javadoc to)